db/reviewsMeta.js

```diff
@@ -0,0 +1,48 @@
+const psql = require('./connector');
+
+const getMetaData = (productId, callback) => {
+  const metaData = {
+    product_id: productId,
+  };
+
+  psql.query(`SELECT rating, COUNT (rating) FROM reviews WHERE product_id = ${productId} GROUP BY rating ORDER BY rating ASC`)
```

Comment:
You might want to parameterize your variables to prevent SQL injection. 
https://node-postgres.com/features/queries

Reply:
I was able to make a new PR with this implementation. thank you for the recommendation